packages/beacon-node/src/metrics/utils/histogram.ts

```diff
@@ -1,48 +0,0 @@
-import {Histogram, HistogramConfiguration} from "prom-client";
-import {IHistogram} from "../interface.js";
-
-type Labels<T extends string> = Partial<Record<T, string | number>>;
-
-/**
- * Extends the prom-client Histogram with extra features:
- * - Add multiple collect functions after instantiation
- * - Create child histograms with fixed labels
- */
-export class HistogramExtra<T extends string> extends Histogram<T> implements IHistogram {
```

Comment:
All this did was add child metric but this should not be required (same as GaugeExtra)